README.md

```diff
@@ -26,21 +26,21 @@ ingress:
   enabled: false
   hosts:
     - service.{{ssm "/exists/subdomain" }}
-    - service1.{{ssm "/empty/subdomain" "required=false" }}
-    - service2.{{ssm "/exists/subdomain" "required=false" "region=eu-west-1" }}
-    - service3.{{ssm "/subdomain" "required=false" "region=eu-west-1" "prefix=/empty" }}
+    - service1.{{ssm "/empty/subdomain" "default=false" }}
```

Comment:
Can you change the default value to something that matches the example?
It's a bit confusing having a boolean as the default for a subdomain